etc/env/example.env

```diff
@@ -165,12 +165,12 @@ KEYCLOAK_REALM=candig
 KEYCLOAK_CLIENT_ID=local_candig
 KEYCLOAK_LOGIN_REDIRECT_PATH=/auth/login
 KEYCLOAK_PORT=8080
+KEYCLOAK_CONTAINER_PORT=8080
```

Comment:
Why do we need a separate entry for the container port?

Reply:
It was added on prod, thought we need it